src/features/island/buildings/components/building/BuildingImageWrapper.tsx

```diff
@@ -1,25 +1,152 @@
+import React, { useState } from "react";
 import classNames from "classnames";
-import React from "react";
+import { InnerPanel } from "components/ui/Panel";
+import { Bar } from "components/ui/ProgressBar";
+import { PIXEL_SCALE } from "features/game/lib/constants";
+import { InventoryItemName } from "features/game/types/game";
+import { ITEM_DETAILS } from "features/game/types/images";
+import { useIsMobile } from "lib/utils/hooks/useIsMobile";
+import { secondsToString } from "lib/utils/time";
+import expressionAlerted from "assets/icons/expression_alerted.png";
 
+/**
+ * BuildingImageWrapper props
+ * @param nonInteractible if the building is non interactable
+ * @param crafting if the building is in the process of crafting
+ * @param ready if the building crafting process is ready
+ * @param itemName the item name currently crafting/crafted
+ * @param craftingSeconds the number of seconds needed to craft the item
+ * @param secondsTillReady the number of seconds needed till the crafting is ready
+ * @param onClick on click event
+ */
 interface Props {
   nonInteractible?: boolean;
+  crafting?: boolean;
+  ready?: boolean;
+  itemName?: InventoryItemName;
+  craftingSeconds?: number;
+  secondsTillReady?: number;
   onClick: () => void;
 }
 
 export const BuildingImageWrapper: React.FC<Props> = ({
   nonInteractible,
+  crafting,
+  ready,
+  itemName,
+  craftingSeconds,
+  secondsTillReady,
   onClick,
   children,
 }) => {
+  const [showHover, setShowHover] = useState(false);
+  const [isMobile] = useIsMobile();
+
+  const onMouseEnter = () => {
+    true && setShowHover(true);
+  };
+
+  const onMouseLeave = () => {
+    setShowHover(false);
+  };
+
+  // time to complete popup
+  // only shows on PC on hover
+  const TimeToComplete = () => {
```

Comment:
We aren't massive fans of the hover information panels. The ux of these on mobile is not good and they can become annoying when you have a very crowded land where these are popping up a lot. I think we would prefer to not add another one in here. 

<img width="355" alt="Screen Shot 2022-11-28 at 12 25 07 pm" src="https://user-images.githubusercontent.com/17863697/204173625-6857f1a4-433e-4523-9814-7cd8073f3d69.png">

Reply:
This has been removed in the recent commits